src/index.html

```diff
@@ -4,13 +4,28 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bar-large">
+      <form class="large-search" data-qa="big">
+        <input class="large-search-bar"
+         type="search"
+         data-qa="keypress"
+         placeholder="Try &quot;Los Angeles&quot;">
```

Comment:
```suggestion
        <input 
           class="large-search-bar"
           type="search"
           data-qa="keypress"
           placeholder="Try &quot;Los Angeles&quot;"
         >
```
pls check your code formatting. fix everywhere

Reply:
це не допомогло, нажаль все так само 1й тест проходить інші 5 ні

Reply:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/143609932/83d0a3cb-9ba7-4d44-9602-057ac48430b5)